config/credentials.default.coffee

```diff
@@ -1,3 +1,5 @@
+secret = () -> Math.random().toString(36).slice(-16)
```

Comment:
it is better have this secret a fixed string, since each call the configure will result in different keys for "secrets" and that could break clustering of kite-user services, since each one will have diff secret for jwt